### PR TITLE
Fixes #20682 - correct usage of pluck

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
also correct a test to catch this.

**Comment by parthaa** [comment]
apj

File: app/models/katello/concerns/host_managed_extensions.rb [hunk 1]
```diff
@@ -75,7 +75,7 @@ def import_package_profile(simple_packages)
 
       def import_package_profile_in_bulk(simple_packages)
         nvras = simple_packages.map { |sp| sp.nvra }
-        found = InstalledPackage.where(:nvra => nvras).pluck(:id, :nvra)
+        found = InstalledPackage.where(:nvra => nvras).select(:id, :nvra)
```
Comment (beav) [inline app/models/katello/concerns/host_managed_extensions.rb]: what was the issue with the old code? I thought getting the non-AR items would be OK here

Comment (jlsherrill) [reply app/models/katello/concerns/host_managed_extensions.rb]: the problem is down below its assuming that its an AR item, calls to .nvra

**Comment by beav** [comment]
apj from me as well

🎸

**Pull request merged**